myscripts.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
+function openNav() {
+  document.getElementById('myLinks').style.width = '100%';
+  document.getElementById('myLinks').style.display = 'block';
+}
+// eslint-disable-next-line no-unused-vars
+function closeNav() {
+  document.getElementById('myLinks').style.width = '0%';
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly adjust your styling to match the Figma design as much as you can below:
*your design*
![notmatch](https://user-images.githubusercontent.com/59397159/173668389-18e4c8b4-7fe3-472b-b233-1f50f4d466ba.png)
*Figma design*
![hey](https://user-images.githubusercontent.com/59397159/173668393-e09d6367-f389-49dd-8f1e-7e402197e392.png)

**Please, kindly do the following:**
- Use the same font weight, font size, font family, and background color.
- Use the correct color for each line after each link.
- Align the links correctly to match the design.
- Align the cancel button correctly to match the design.
- The mobile menu container should take the whole device width and height so that the user must click on any link to view that section on your page. As of now, the user may choose to ignore the links and just scroll down the page, which I think is not what you want to achieve and also to improve the user experience.
- Add the missing white background and black line as per the design to match it as much as you can

Reply:
thanks and i have resolved it